packages/lesswrong/server/callbacks/alignment-forum/callbacks.ts

```diff
@@ -72,20 +46,15 @@ async function updateAlignmentUserServer (newDocument: DbVoteableType, vote: DbV
   }
 }
 
-export async function updateAlignmentUserServerCallback ({newDocument, vote}: VoteDocTuple) {
-  await updateAlignmentUserServer(newDocument, vote, 1)
+export async function grantUserAFKarmaForVote ({newDocument, vote}: VoteDocTuple) {
+  await updateUserAFKarmaForVote(newDocument, vote, 1)
 }
 
-export async function cancelAlignmentUserKarmaServer ({newDocument, vote}: VoteDocTuple) {
-  await updateAlignmentUserServer(newDocument, vote, -1)
+export async function revoteUserAFKarmaForCancelledVote ({newDocument, vote}: VoteDocTuple) {
```

Comment:
should this be `revoke` rather than `revote`?

Reply:
Yep, typo